os-hardening/minimal_access.sls

```diff
@@ -1,7 +1,7 @@
 {% from "os-hardening/map.jinja" import hardening with context %}
-{% for dir in hardening.read_only_folders %}
-{{dir}}:
+hardening-read-only-folders:
   file.directory:
+    - names: {{ hardening.read_only_folders }}
```

Comment:
hmm my bad, my assumption was names is global to all state functions 

--
Puneet

> On Apr 4, 2015, at 16:45, Nitin Madhok <notifications@github.com> wrote:
> 
> In os-hardening/minimal_access.sls:
> 
> >    file.directory:
> > +    - names: {{ hardening.read_only_folders }}
> names does not exist as an argument to file.directory: http://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.directory
> 
> —
> Reply to this email directly or view it on GitHub.
>